package.json

```diff
@@ -10,7 +10,7 @@
   "engines": {
     "vscode": "^1.55.0"
   },
-  "icon": "images/daffodil.jpg",
+  "icon": "images/daffodil.png",
```

Comment:
Should this be an actual icon file, or a standard square shape? Does this icon show up some where in VS Code?

We do have an actual .ico file used for the Windows MSI that we could reuse for this if an .ico file makes more sense than the log:

https://raw.githubusercontent.com/apache/daffodil/main/daffodil-cli/src/windows/apache-daffodil.ico

Reply:
So this icon will appear when you install extension here is a screenshot of mine with the current image

![Screen Shot 2022-02-18 at 11 46 54 AM](https://user-images.githubusercontent.com/32347414/154726047-a0f5fb11-b807-420b-a819-b9afd7ef3254.png)

This is with the image you linked

![Screen Shot 2022-02-18 at 11 50 24 AM](https://user-images.githubusercontent.com/32347414/154726606-8bb3bb62-b033-44c8-8f1b-0a5759de8d3a.png)

Which do you prefer? I personally prefer the image you linked.

Reply:
Yeah, I think the .ico looks better. I also found the [VS Code package.json documentation](https://code.visualstudio.com/api/references/extension-manifest) that says the icon property is:

> The path to the icon of at least 128x128 pixels (256x256 for Retina screens).

Doesn't say anything about the supported formats, but it looks like the .ico  works for you. And I believe the .ico file has a bunch of built-in resolutions, ranging from 16x16 to 256x256, so should scale nicely if VS Code uses it anywhere else.